test/tests-crud-hooks.js

```diff
@@ -616,7 +616,7 @@ spawnedTest("updating using Table.bulkPut()", function*(){
         op: "update",
         key: 1,
         obj: {id:1, address: {city: 'A'}},
-        mods: {"address.city": "B"},
+        mods: {address: {city: "B"}},
```

Comment:
What is the reason `{"address.city": "B"}` is changed to `{address: {city: "B"}}`? In the original code, we are just replacing the "city" property of the inner object, and that must also be possible to do. The changed mods means we are replacing the whole "address" property to be exactly {city: "B"} which is different.

It is important that the existing unit tests (with `{"address.city": "B"}`) are still working so it would be better to leave existing unit tests as they are.